packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -236,6 +237,23 @@ export const FLAGS: Record<string, FlagInfo> = {
     compatibilityWithOldBehaviorMd: 'You can pass `desiredCount: 1` explicitly, but you should never need this.',
   },
 
+  //////////////////////////////////////////////////////////////////////
+  [ECS_ADD_SECURITY_GROUP]: {
+    type: FlagType.ApiDefault,
+    summary: 'Add security group through "configureAutoScalingGroup"',
+    detailsMd: `
+    ConfigureAutoScalingGroup, currently determine for the ecs security group  of
+    a CfnService if a security group is not provided. The result of this is that on every
+    deployment, the security group is reset to the fixed, even if it was autoscaled.
+    
+    If this flag is not set, the cluster.addAutoScalingGroup adds security groups from 
+    the ASG's connections into the ECS Cluster's connections, but cluster.addAsgCapacityProvider does not `,
+    introducedIn: { v1: '1.92.0' },
```

Comment:
Please read the feature flag guidelines [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#feature-flags) and ensure this PR meets them. For example, you need to be careful that this `introducedIn` field is correct. It should be `{ v2: 'V2NEXT' }`.

Also, make sure the defaults are right. Basically please be sure that this is exactly correct, it's very important to be precise here.

Reply:
changed 
```
introducedIn: { v2: 'V2NEXT' },
    defaults: { v2: true },
    recommendedValue: true,
```